lib/hopper/lazy_source.rb

```diff
@@ -13,9 +13,9 @@ def __target_object__
   end
 
   # rubocop disable is needed as the class is extending BasicObject (no respond_to_missing?, should not respond to super)
-  # rubocop:disable Style/MissingRespondToMissing, Lint/MissingSuper
+  # rubocop:disable Style/MissingRespondToMissing
```

Comment:
I feel like we fixed this before, remember commenting on the stupid escape

Reply:
I missed hopper, porky, daffy, token_validator during my rubocop update 😬